packages/backend/discovery/shared-eigenlayer/ethereum/config.jsonc

```diff
@@ -47,7 +47,7 @@
   },
   "overrides": {
     "TokenHopper": {
-      "ignoreInWatchMode": ["latestPress"]
```

Comment:
can you add the description here?

Reply:
fixded, forgot about it and thank you for noticing